mnefun/_mnefun.py

```diff
@@ -980,12 +991,11 @@ def run_sss_locally(p, subjects, run_indices):
                 raw = Raw(r, preload=True, allow_maxshield=True)
                 raw.load_bad_channels(prebad_file, force=True)
                 # apply maxwell filter
-                raw_sss = maxwell_filter(raw, int_order=p.int_order, ext_order=p.ext_order,
-                                         calibration=cal_file,
-                                         cross_talk=ct_file,
-                                         st_correlation=p.st_correlation, st_duration=st_duration,
-                                         destination=None,
-                                         coord_frame='meg')
+                raw_sss = maxwell_filter(
+                    raw, int_order=p.int_order, ext_order=p.ext_order,
+                    calibration=cal_file, cross_talk=ct_file,
+                    st_correlation=p.st_correlation, st_duration=st_duration,
+                    destination=None, coord_frame='meg')
                 raw_sss.save(o, overwrite=True, buffer_size_sec=None)
 
```

Comment:
Yeah, PEP8 checking is in effect and I am not getting any notifications that lines are running long. I have noticed that between pycharm, atom, and spyder editors the character demarcation results in shorter lines in e.g. atom vs. pycharm. I'll look into what pycharm is doing.